src/components/DpDataTable/DpTableRow.vue

```diff
@@ -33,7 +33,7 @@
     <td
       v-for="(field, idx) in fields"
       :key="`${field}:${idx}`"
-      :class="{ 'c-data-table__resizable': isTruncatable } + ' overflow-hidden'"
+      :class="{ 'c-data-table__resizable': isTruncatable }"
```

Comment:
did You check why it was added? I am not sure, if we simply can remove it without re-adding other issues.

Reply:
Yes, see second point above in PR description

Reply:
> Because the columns are resizable and the user can easily adjust the column to make more space for the menu

I would argue against that. When I want to access the Navigation quickly, I don't want to slide the cols "all the time".
Can we leave it like it was and try to find a overarching solution afterwards?

Reply:
You don't need to do it all the time, only once. The width of the columns is saved. On the other hand, with overflow-hidden the user can't access the dropdown in selects if the the table row is not big enough and can't use the feature. I agree, that an overarching solution would be nice, but I could'nt find one so far and imo this is a much bigger problem than the orange bug that was fixed with this.

Reply:
:thinking:

Reply:
I would love to put an explaining comment somewhere, but I don't know where. I see Your point. I didn't think about interactive elements within the cell-content.